packages/discovery/Discovery.test.ts

```diff
@@ -49,7 +50,12 @@ describe('Discovery', () => {
 
         const valid = validate(discovery);
         if (!valid) {
-            throw new Error('Invalid discovery file:' + JSON.stringify(validate.errors, null,  2));
+            const output = betterAjvErrors(DiscoverySchema, discovery, validate.errors!, {
+                format: 'cli',
+                indent: 2
+            });
+            console.log(output);
```

Comment:
@josejulio was this `console.log` added by design or did you use it for local testing?

Reply:
It's by design, it will output what is show in the screenshot - thanks for checking!

Reply:
Awesome!